SofaKernel/modules/SofaImplicitOdeSolver/src/SofaImplicitOdeSolver/NBESolver.h

```diff
@@ -0,0 +1,43 @@
+#ifndef VNCS_NBESOLVER_H
+#define VNCS_NBESOLVER_H
```

Comment:
```suggestion
#pragma once
```
and remove the `#endif` at the end of the file